packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -223,10 +223,7 @@ const DropdownMenuRootContent = React.forwardRef((props, forwardedRef) => {
       disableOutsideScroll={disableOutsideScroll}
       portalled={portalled}
       trapFocus
-      onCloseAutoFocus={composeEventHandlers(props.onCloseAutoFocus, (event) => {
-        event.preventDefault();
-        context.triggerRef.current?.focus();
-      })}
+      closeAutoFocusRef={context.triggerRef}
```

Comment:
@benoitgrelard I'm not sure how I feel about this. 

This is making me wonder why we wouldn't have an `openAutoFocusRef` also now so things like `AlertDialog` wouldn't need to `event.preventDefault()` in `onOpenAutoFocus`, they'd just pass a node.

That's then making me wonder why we need the `onCloseAutoFocus` and `onOpenAutoFocus` events at all....

Is this PR the right approach or is there another way we could solve the original issue? I'm still so full of brain fog 🙈

Reply:
That would be going back to a direction we had originally and that's what most libs do, but I feel like we moved away for a reason. I do like the approach we have that we "do the thing" and people can prevent that and do their own thing instead.

For the original issue, we could simply do that? prevent default and specifically focus the trigger?

Reply:
The original issue happens because when focus scope gets `document.activeElement` it is the `input` inside the focus scope because the `autoFocus` attribute focuses it before the `FocusScope` effect runs. So, when the focus scope closes, it tries to focus the input from inside content, but it has been unmounted.

> For the original issue, we could simply do that? prevent default and specifically focus the trigger?

That's true. I guess I thought we needed this change so all the other components would work if something was `autoFocus`ed inside them but I forgot we already do this `event.preventDefault()` thing in those components so it wouldn't be a problem there. DOH! Not sure why I didn't think of that, will give it a go!